chirp/drivers/kguv9dplus.py

```diff
@@ -1263,73 +1577,75 @@ def _core_tab(self):
         Radio settings common to all modes and areas go here.
         """
         s = self._memobj.settings
+        if self.MODEL == "KG-UV9PX":
+            sm = self._memobj.screen
```

Comment:
Not critical, but it seems like it would be better to put this down near where it's used, no?

Reply:
agree... will update it in a future clean-up